src/open_inwoner/cms/utils/plugin_mixins.py

```diff
@@ -0,0 +1,14 @@
+from cms.models import Page
+from cms.plugin_base import CMSPluginBase
+
+
+class CMSActiveAppMixin:
+    app_hook = None
+
+    @property
+    def render_plugin(self):
+        return Page.objects.published().filter(application_urls=self.app_hook).exists()
```

Comment:
I think this should raise an error if `app_hook` is `None`, just to be safe.

Not sure what type, maybe ValueError? It doesn't really matter as this is mostly to catch development issues.